src/makeInfinityAdder.js

```diff
@@ -4,7 +4,21 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  return function adder(num) {
+    if (num === undefined) {
+      const resultat = sum;
+
+      sum = 0;
+
+      return resultat;
+    }
+
+    sum += num;
```

Comment:
Ensure that `num` is a number before adding it to `sum`. You might want to add a check to handle non-numeric inputs gracefully, such as throwing an error or ignoring the input.